ui/src/app/components/help-sidebar-icons.tsx

```diff
@@ -105,7 +111,8 @@ export type SidebarIconId =
   | 'annotations'
   | 'apps'
   | 'runtimeConfig'
-  | 'cromwellConfig'
+  | typeof cromwellConfigIconId
```

Comment:
TypeScript tries to be as specific as possible with its types, including leveraging the actual value of a variable if possible. In this case, the type of `cromwellConfigIconId` is actually just the string `'cromwellConfig'`. Because it's declared with `const`, we can never change it to another string.

Minimal example:

<img width="798" alt="image" src="https://user-images.githubusercontent.com/31020403/232134102-c1d2d73b-ab71-4af7-a0be-f411e3a65898.png">

Reply:
Interesting! I didn't know that. Thanks for explaining.